feathr_project/test/test_input_output_sources.py

```diff
@@ -36,7 +38,7 @@ def test_feathr_get_offline_features_with_parquet():
     now = datetime.now()
     # set output folder based on different runtime
     if client.spark_runtime == 'databricks':
-        output_path = ''.join(['dbfs:/feathrazure_cijob','_', str(now.minute), '_', str(now.second), ".parquet"])
+        output_path = ''.join(['dbfs:/feathrazure_cijob','_', str(now.minute), '_', str(now.second),'_', str(now.microsecond),  ".parquet"])
```

Comment:
Is there any specifc reason only add this for databricks

Reply:
In very rare cases databricks will fail to read the result file (though the spark job is successful). Assuming it might be related with the output folder write conflict. This might apply to Synapse as well but I never see that in Synapse so didn't add it there.